packages/flutter/lib/src/material/checkbox_list_tile.dart

```diff
@@ -170,7 +170,7 @@ class CheckboxListTile extends StatelessWidget {
   /// ```dart
```

Comment:
To ensure that we never break this example code again, it would be best to wrap this in a `{@tool snippet}` section similarly to how it is done here: https://github.com/flutter/flutter/blob/8560d914eb8bc965306dfb9336425cac19670358/packages/flutter/lib/src/animation/animation.dart#L109

That way, this example gets analyzed by the analyzer and doesn't break that easily. Would you be up for refactoring that?